plugins/sms-twilio/src/manifest.json

```diff
@@ -166,6 +166,12 @@
       "label": "service.suspend",
       "description": "Example: Suspended service %%service.name%% for client %%client.companyName%%%%client.firstName%% %%client.lastName%%.",
       "required": 0
+    },
+    {
+      "key": "logging_level",
+      "label": "set logging level",
+      "description": "Make the plugin more or less verbose. Available levels: emergency, alert, critical, error, warning, notice, info (default), debug",
```

Comment:
This is unnecessarily complex, we only use error, warning, info and debug. As you always want to see errors and warnings, the only things left are info and debug messages. I'd argue you'd always want info as well, so IMO this can be simplified to just a checkbox "allow debug logging".